ykrt/src/compile/jitc_yk/opt/mod.rs

```diff
@@ -118,6 +118,17 @@ impl Opt {
                         self.m.replace(iidx, Inst::Const(dst_cidx));
                     }
                 }
+                Inst::LoadTraceInput(x) => {
```

Comment:
I'm not sure what this last hunk is for. Maybe a comment would help?

Reply:
The github issue in the comment after this line hopefully explains things. Ultimately this clause shouldn't really exist.

Reply:
I see. Yeah, we should look into this at some point (but not now).